src/feature/Layout/Layout.jsx

```diff
@@ -0,0 +1,28 @@
+import { useGetUser as getUser } from "data-access/useGetUser";
+import { Footer } from "../Footer";
+import { NavigationBar } from "../NavigationBar";
+import { useEffect, useState } from "react";
+
+export const Layout = ({ children }) => {
+  const [userEmail, setUserEmail] = useState();
+  const [userImgSource, setUserImgSource] = useState();
+
+  const handleLoadUser = async () => {
+    const { data } = await getUser();
+    const { email, image_source } = data[0] || {};
+    setUserEmail(email);
+    setUserImgSource(image_source);
+  };
+
+  useEffect(() => {
+    handleLoadUser();
+  }, []);
+
+  return (
+    <div>
+      <NavigationBar userEmail={userEmail} userImgSource={userImgSource} />
+      <main>{children}</main>
+      <Footer />
+    </div>
+  );
```

Comment:
P2:
유저정보니 하나로 관리해도 좋을 것 같아요. 나눴을 때 이점을 잘 모르겠어서요.

```suggestion
  const [userInfo, setUserInfo] = useState();
  
  const handleLoadUser = async () => {
    const { data } = await getUser();
    const { email, image_source } = data[0] || {};
    
    setUserInfo({ email, image: image_source });
  };

  useEffect(() => {
    handleLoadUser();
  }, []);

  return (
    <div>
      <NavigationBar {...userInfo} /> // 만약 이렇게 작성하시려면 NavigationBar에서 userInfo에서 넘기는 데이터와 동일한 key값을 props로 받아야 합니다. 여기서는 email, image가 되겠네요.
      <main>{children}</main>
      <Footer />
    </div>
  );
```